content/millennium/r4/base/entities/location.md

```diff
@@ -11,15 +11,24 @@ title: Location | R4 API
 
 The Location resource describes physical places where healthcare services are provided. In Millennium, facilities are the top level of the patient location hierarchy. A facility is also an Organization resource at which patient locations are associated. All facilities are organizations, but not all organizations are facilities. A location can also be an ambulatory patient care area like a clinic or an emergency room. The location hierarchy from highest to lowest is facility, building, nursing unit, room, and bed.
 
+* The following [HL7® FHIR® US Core Implementation Guide STU 4.0.0](https://hl7.org/fhir/us/core/STU4/){:target="_blank"} Profiles are supported by this resource:
+
+  * [US Core Location Profile](http://hl7.org/fhir/us/core/STU4/StructureDefinition-us-core-location.html){:target="_blank"}
+
 The following fields are returned if valued:
 
 * [Location id](http://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
 * [Status](http://hl7.org/fhir/R4/location-definitions.html#Location.status){:target="_blank"}
+  * [Location Status](http://hl7.org/fhir/R4/valueset-location-status.html){:target="_blank"} ([Active](http://hl7.org/fhir/R4/codesystem-location-status.html#location-status-active){:target="_blank"} \| [Inactive](http://hl7.org/fhir/R4/codesystem-location-status.html#location-status-inactive){:target="_blank"})
```

Comment:
The Location Status, Mode, and Telecom changes should be documented in the Terminology Bindings section and not in the supported fields list.

To do that, add the appropriate details [here](https://github.com/cerner/fhir.cerner.com/blob/db9f6bd5ad3c94738a19df8c1c2bc0ce98c897c1/lib/resources/r4/location.yaml) and they will be pulled into the existing Terminology Bindings table

Reply:
Solved in [ce62e40](https://github.com/cerner/fhir.cerner.com/pull/887/commits/ce62e40f81fa6174d4a9f7b8d9b45d7089d3d3f7)